### PR TITLE
Lamp Bars

### PR DESCRIPTION
**Pull request by Supermichael777**
Adds lamp bars to the game. They glow and have a chemical that makes moths act a bit funny.

### CODE REVIEW

File: code/game/objects/items/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -2724,6 +2724,19 @@
 		icon_state = "[initial(icon_state)]-o"
 		package = FALSE
 
+/obj/item/reagent_container/food/snacks/wrapped/lampbar  //LAMP!
+	name = "LAMP Bar"
+	desc = "A glow in the dark candybar. Made with real glowshrooms. <B>\"LAMP!\"</B>"
```
Comment (Rohesie) [inline code/game/objects/items/reagent_containers/food/snacks.dm]:
```suggestion
	desc = "A glow-in-the-dark candybar. Made with real glowshrooms. <B>\"LAMP!\"</B>"
```

File: code/modules/reagents/chemistry_reagents/other.dm [hunk 2]
```diff
@@ -221,6 +221,20 @@
...
+/datum/reagent/mothpheromones
+	name = "Moth pheromones"
+	id = "mothpheromones"
+	description = "Basically the liquid concept of fluttering towards a lamp."
+	reagent_state = LIQUID
+	color = "#8fddf7" // rgb: 143, 221, 247
+	taste_description = "silk"
+
+/datum/reagent/mothpheromones/on_mob_life(mob/living/M, alien)
+	if(alien == IS_MOTH)
```
Comment (Rohesie) [inline code/modules/reagents/chemistry_reagents/other.dm]:
```suggestion
	if(alien == IS_MOTH && prob(7))
```

File: code/modules/reagents/chemistry_reagents/other.dm [hunk 3]
```diff
@@ -221,6 +221,20 @@
...
+		if(prob(7))
```
Comment (Rohesie) [inline code/modules/reagents/chemistry_reagents/other.dm]:
```suggestion

```

File: code/modules/reagents/chemistry_reagents/other.dm [hunk 4]
```diff
@@ -221,6 +221,19 @@
...
+	name = "Moth pheromones"
+	id = "mothpheromones"
+	description = "Basically the liquid concept of fluttering towards a lamp."
+	reagent_state = LIQUID
+	color = "#8fddf7" // rgb: 143, 221, 247
+	taste_description = "silk"
+
+/datum/reagent/mothpheromones/on_mob_life(mob/living/M, alien)
+	if(alien == IS_MOTH && prob(7))
+		M.emote(pick("twitch","drool","moan","gasp"))
```
Comment (Qustinnus) [inline code/modules/reagents/chemistry_reagents/other.dm]: remove moan

Comment (Supermichael777) [reply code/modules/reagents/chemistry_reagents/other.dm]: No

Comment (Qustinnus) [reply code/modules/reagents/chemistry_reagents/other.dm]: that makes this easy then

Comment (Rohesie) [reply code/modules/reagents/chemistry_reagents/other.dm]: Remove the moan for Qust to approve it, so we have nice lamp bars, please.

**Pull request closed without merging**